EssentialsDiscordLink/src/main/java/net/essentialsx/discordlink/listeners/LinkBukkitListener.java

```diff
@@ -52,7 +71,12 @@ public void onConnect(final AsyncPlayerPreLoginEvent event) {
             return;
```

Comment:
The nickname synchronization code will not run due to this `return` if the link policy isn't set to kick unlinked players